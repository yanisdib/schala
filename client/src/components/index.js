// COMPONENTS EXPORTS

export { default as Header } from './Header/Header';

export { default as Button } from './Button/Button';
export { default as MenuButton } from './MenuButton/MenuButton';
export { default as DisplayIdentifiedButton } from './DisplayIdentifiedButton/DisplayIdentifiedButton';

export { default as Tag } from './Tag/Tag';
export { default as Pointer } from './Pointer/Pointer';
export { default as Bubble } from './Bubble/Bubble';

export { default as PictureIdentifier } from './PictureIdentifier/PictureIdentifier';