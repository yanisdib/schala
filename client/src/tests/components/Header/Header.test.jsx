import { shallow } from 'enzyme';

import { Header } from '../../../components';


describe('Header component', () => {
    let wrapper;
    let defaultProps = Header.defaultProps.isLogged;

    beforeEach(() => wrapper = shallow(<Header />));

    it('renders properly', () => {
        expect(wrapper).toMatchSnapshot();
    });

    it('renders properly with props as true', () => {
        wrapper = shallow(<Header isLogged={true} />);
        expect(wrapper).toMatchSnapshot();
    });

    it('renders properly with default props', () => {
        expect(defaultProps).toBeFalsy();
    });
})