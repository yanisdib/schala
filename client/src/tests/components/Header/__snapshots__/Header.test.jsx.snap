// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component renders properly 1`] = `
<header>
  <div
    className="header-wrapper"
  >
    <div
      className="logo"
    />
    <div
      className="signup-button"
    >
      <styled.button
        variant="default"
      >
        Signup
      </styled.button>
    </div>
    <div
      className="signin-button"
    >
      <styled.button
        variant="login"
      >
        Login
      </styled.button>
    </div>
    <div
      className="header-menu-button"
    >
      <MenuButton
        onClick={[Function]}
      />
    </div>
  </div>
</header>
`;

exports[`Header component renders properly with props as true 1`] = `""`;
